glide.lock

```diff
@@ -223,7 +223,7 @@ imports:
   - transport
 - name: gopkg.in/src-d/go-errors.v0
   version: 0c303ec4c027302259ec1738f19f515aad25f13f
-- name: gopkg.in/src-d/simple-linguist.v1
+- name: gopkg.in/src-d/enry.v1
```

Comment:
Stupid question: Shouldn't we use `github.com/src-d/enry` instead now that we are versioning using commits?

Now that we are using glide, what we want is to use a particular commit from `github.com/src-d/enry`, not from `gopkg.in/src-d/enry.v1`, right?